### ISSUE TITLE
Correctly camel case kebab-case path

### ISSUE THREAD

**Issue by hanayashiki** [anymaniax/orval#340]
## What are the steps to reproduce this issue?
My `petstore.yaml`

```yaml
basePath: /api/v1
host: 127.0.0.1:8000
info:
  contact: {}
  description: Foobar
  title: Foobar
  version: "1.0"
paths:
  /foo-bar/:
    get:
      description: Showcase that orval can't correctly camel-case kebab-case path name.
      produces:
      - application/json
      responses:
        "200":
          description: OK
          schema:
            type: string
      summary: foo bar
      tags:
      - FooBar
swagger: "2.0"

```

My orval.config.ts

```
module.exports = {
  internal_api: {
    output: {
      mode: 'tags-split',
      target: '.internal_api/apis.ts',
      schemas: '.internal_api/models',
      client: 'react-query',
      mock: false,
      clean: true,
    },
    input: {
      target: './petstore.yaml',
    },
  },
};

```

## What happens?

Generated `foo-bar.ts` names the hook as `useGetFoobar`, instead of `useGetFooBar`

```
/**
 * Generated by orval v6.6.4 🍺
 * Do not edit manually.
 * Foobar
 * Foobar
 * OpenAPI spec version: 1.0
 */
import axios,{
  AxiosRequestConfig,
  AxiosResponse,
  AxiosError
} from 'axios'
import {
  useQuery,
  UseQueryOptions,
  QueryFunction,
  UseQueryResult,
  QueryKey
} from 'react-query'

// eslint-disable-next-line @typescript-eslint/no-explicit-any
type AsyncReturnType<
T extends (...args: any) => Promise<any>
> = T extends (...args: any) => Promise<infer R> ? R : any;


/**
 * Showcase that orval can't correctly camel-case kebab-case path name.
 * @summary foo bar
 */
export const getFoobar = (
     options?: AxiosRequestConfig
 ): Promise<AxiosResponse<string>> => {
    return axios.get(
      `/foo-bar/`,options
    );
  }


export const getGetFoobarQueryKey = () => [`/foo-bar/`];

    
export const useGetFoobar = <TData = AsyncReturnType<typeof getFoobar>, TError = AxiosError<unknown>>(
  options?: { query?:UseQueryOptions<AsyncReturnType<typeof getFoobar>, TError, TData>, axios?: AxiosRequestConfig}

  ):  UseQueryResult<TData, TError> & { queryKey: QueryKey } => {

  const {query: queryOptions, axios: axiosOptions} = options || {}

  const queryKey = queryOptions?.queryKey ?? getGetFoobarQueryKey();

  

  const queryFn: QueryFunction<AsyncReturnType<typeof getFoobar>> = () => getFoobar(axiosOptions);

  const query = useQuery<AsyncReturnType<typeof getFoobar>, TError, TData>(queryKey, queryFn, queryOptions)

  return {
    queryKey,
    ...query
  }
}


```

## What were you expecting to happen?

The name associated with `foo-bar` are converted to `FooBar` instead of `Foobar`.

## Any logs, error output, etc?

Above

## Any other comments?

No

## What versions are you using?
**Operating System:**  ...
**Package Version:** 6.6.4
**Browser Version:** …

**Comment by anymaniax** [comment]
will be fix in the next version

**Comment by anymaniax** [comment]
Fixed with `6.7.0`
